README.md

```diff
@@ -0,0 +1,140 @@
+## SDK Features
+
+This repository contains snippets for many Temporal SDK features written in different Temporal SDK languages. This also
+contains a runner and language-specific harnesses to confirm feature behavior across versions.
+
+These SDK features serve several purposes:
+
+* Ensure parity across SDKs by having same-feature snippets adjacent to one another
+* Confirm feature behavior across SDK versions
+* Confirm history across SDK versions
+* Document features in different SDKs
+* Easy-to-use environment for writing quick workflows in all languages/versions
+
+## Building
+
+With latest [Go](https://golang.org/) installed, run:
+
+    go build
+
+## Running
+
+Prerequisites:
+
+* [Go](https://golang.org/) 1.17+
+* [JDK](https://adoptium.net/?variant=openjdk11&jvmVariant=hotspot) 11+
+* [Node](https://nodejs.org) 16+
```

Comment:
The SDK can run with node 14, we might even want to run the tests with multiple node versions for extra assurance.

Reply:
Yeah, we'll have to look into advanced cases of different computers or runtimes. Currently I don't support dynamically changing the Go, Java, or Node versions (nor do I support changing versions of the embedded server yet).

But I don't think that's needed for this PR

Reply:
I guess this could be added to the TODOs section below

Reply:
Added